Cloudinary/Features/Helpers/Layers/CLDTextLayer.swift

```diff
@@ -351,4 +409,20 @@ import Foundation
         
         return properties
     }
+    
+    public enum CLDFontAntialiasing: String {
```

Comment:
Please add documentation string for each option.
Can be taken from the docs site - https://cloudinary.com/documentation/image_transformations#adding_text_captions